changelog.md

```diff
@@ -1,5 +1,26 @@
 
 
+## 2.6.3 / 2020-04-09
+
+### Improvement
+
+- Database index improvement to increase Tracim reactivity (#2787)
+
+### Fixed issues
+
+- Frontend: #2688
+- Backend: #2687, #2793, #2811, #2812
+
+### Other Changes
+
+- ElasticSearch: Refactor of indexing organization. So its necessary to drop existing index and create it again to use ElasticSearch, use CLI command for this (#2660)
+
+### Known issues
```

Comment:
```suggestion
### Known Issues
```

Reply:
Done and change all `issues` to `Issues`